js-intro.js

```diff
@@ -5,53 +5,113 @@
 
 var mantra = "Be the dev"
 
-// 1a. Write the code that determines if there is a 'B' in mantra.
+// // 1a. Write the code that determines if there is a 'B' in mantra.
 
+var lowerMantra = mantra.toLowerCase()
 
+if(lowerMantra.includes("b")) {
+    console.log("'B' is in mantra.")
+} else {
+    console.log("'B' is not in mantra.")
+}
 
-// 1b. Write the code that determines if there is an 'x' in mantra.
+// // 1b. Write the code that determines if there is an 'x' in mantra.
 
+if(lowerMantra.includes("x")) {
+    console.log("'X' is in mantra.")
+} else {
+    console.log("'X' is not in mantra.")
+}
 
+// // 1c. Write the code that determines if there is a 'v' in mantra.
 
-// 1c. Write the code that determines if there is a 'v' in mantra.
-
-
+if(lowerMantra.includes("V")) {
+    console.log("'V' is in mantra.")
+} else {
+    console.log("'V' is not in mantra.")
+}
 
 // 1d. STRETCH: Write a function that can take two arguments - a string and a letter and returns a sentence telling the user if the letter exists within the string.
 
+function letterExistsOrNot(str, ltr) {
+    if (str.includes(ltr)) {
+        console.log(`"${ltr}" is in the string.`)
+    } else {
+        console.log(`"${ltr}" is not in the string.`)
+    }
+}
+
+var str = "You only live once"
+var ltr = "y"
 
+console.log(letterExistsOrNot(str.toLowerCase(), ltr))
 
```

Comment:
This code looks great but rather than have a console log in the function and on the function call, have the string interpolation in a return statement.